python/GafferUI/ColorChooser.py

```diff
@@ -244,23 +245,31 @@ def _displayTransformChanged( self ) :
 
 class _ColorField( GafferUI.Widget ) :
 
-	def __init__( self, color = imath.Color3f( 1.0 ), staticComponent = "v", dynamicBackground = True, **kw ) :
+	__DragConstraints = enum.Flag( "__DragConstraints", [ "X", "Y" ] )
+
+	def __init__( self, color = imath.Color3f( 1.0 ), staticComponent = "v", dynamicBackground = False, **kw ) :
```

Comment:
Somewhere along the way we've ended up with `dynamicBackground = False` here.

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/6128/commits/0a9e991402ee6077020bc4f0e899af1f269e8d1c. Sorry for missing that and thanks for catching it.